### PR TITLE
Added configuration for SSL keystores

### PR DESCRIPTION
**Pull request by MatthiasBechtold**
Code was compiled & tested successfully in my use case.

### CODE REVIEW

File: src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java [hunk 1]
```diff
@@ -240,15 +256,31 @@
                       CONFIG_DOCUMENTATION_MQ_SSL_PEER_NAME, CONFIG_GROUP_MQ, 13, Width.MEDIUM,
                       CONFIG_DISPLAY_MQ_SSL_PEER_NAME);
 
+        config.define(CONFIG_NAME_MQ_SSL_KEYSTORE_LOCATION, Type.STRING, null, Importance.MEDIUM,
+                      CONFIG_DOCUMENTATION_MQ_SSL_KEYSTORE_LOCATION, CONFIG_GROUP_MQ, 14, Width.MEDIUM,
+                      CONFIG_DISPLAY_MQ_SSL_KEYSTORE_LOCATION);
+
+        config.define(CONFIG_NAME_MQ_SSL_KEYSTORE_PASSWORD, Type.STRING, null, Importance.MEDIUM,
```
Comment (AndrewJSchofield) [inline src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java]: Please use Type.PASSWORD so the password is handled as a piece of secret data like CONFIG_NAME_MQ_PASSWORD.

File: src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java [hunk 2]
```diff
@@ -240,15 +256,31 @@
...
+                      CONFIG_DOCUMENTATION_MQ_SSL_KEYSTORE_PASSWORD, CONFIG_GROUP_MQ, 15, Width.MEDIUM,
+                      CONFIG_DISPLAY_MQ_SSL_KEYSTORE_PASSWORD);
+
+        config.define(CONFIG_NAME_MQ_SSL_TRUSTSTORE_LOCATION, Type.STRING, null, Importance.MEDIUM,
+                      CONFIG_DOCUMENTATION_MQ_SSL_TRUSTSTORE_LOCATION, CONFIG_GROUP_MQ, 16, Width.MEDIUM,
+                      CONFIG_DISPLAY_MQ_SSL_TRUSTSTORE_LOCATION);
+
+        config.define(CONFIG_NAME_MQ_SSL_TRUSTSTORE_PASSWORD, Type.STRING, null, Importance.MEDIUM,
```
Comment (AndrewJSchofield) [inline src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java]: Please use Type.PASSWORD so the password is handled as a piece of secret data like CONFIG_NAME_MQ_PASSWORD.

File: src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java (hunk 1 above)
Comment (MatthiasBechtold) [reply src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java]: Sure, that makes sense. Thanks for the review!

**Comment by MatthiasBechtold** [comment]
I hereby accept the CLA associated with this repository.

File: src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java [hunk 3]
```diff
@@ -107,7 +123,7 @@
     public static final String CONFIG_DOCUMENTATION_TOPIC = "The name of the target Kafka topic.";
     public static final String CONFIG_DISPLAY_TOPIC = "Target Kafka topic";
 
-    public static String VERSION = "1.1.0";
+    public static String VERSION = "1.2.0";
```
Comment (AndrewJSchofield) [inline src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java]: I've been reviewing best practice for Maven versioning. This PR by itself will not form v1.2.0. Please use `1.2.0-SNAPSHOT` for all of the versions (you missed in instance in the Dockerfile I think).

Comment (MatthiasBechtold) [reply src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java]: Correct, I missed that. Thanks for the hint!

**Comment by AndrewJSchofield** [comment]
Please resolve the conflicts so it's ready to merge.

**Comment by MatthiasBechtold** [comment]
Done.

**Pull request merged**